templates/set-absolute-and-idle-timeout-based-on-organization-connection-or-application-metadata-POST_LOGIN/manifest.yaml

```diff
@@ -0,0 +1,11 @@
+id: "454ef310-4779-41bb-bed4-c916ab9eaaed"
+name: "Set Absolute and Idle Timeout based on Organization, Connection or Application Metadata"
+triggers:
+		- "POST_LOGIN"
+useCases:
+		- "ACCESS_CONTROL"	 
```

Comment:
just fix the spacing on these two items in this file and I think we're good to go